src/loaddb/load_db.c

```diff
@@ -1194,6 +1199,44 @@ register_signal_handlers ()
   util_arm_signal_handlers (sig_handler, sig_handler);
 }
 
+static int
+load_has_authorization (const std::string & class_name, DB_AUTH au_type)
```

Comment:
Do you want `auth_type`? if the case, `DB_AUTH_INSERT` in line 1230 should be modified.